source/interface/directoryContents.js

```diff
@@ -13,7 +13,7 @@ function getDirectoryContents(remotePath, options) {
         method: "PROPFIND",
         headers: {
             Accept: "text/plain",
-            Depth: 1
+            Depth: options.deep ? "infinity" : 1
```

Comment:
This already exists in master.. perhaps you could merge/rebase your branch?

Reply:
Hopefully I cleaned up the history correctly.  I'm willing to try again if necessary.